UnityProject/Assets/Scripts/MiniGames/IMiniGame.cs

```diff
@@ -0,0 +1,9 @@
+namespace MiniGames
+{
+	public interface IMiniGame
```

Comment:
I don't see this interface used anywhere?

Reply:
It is being used.